src/backend/blockchain/src/routes.py

```diff
@@ -21,9 +50,29 @@ def index():
 def get_chain():
     chain = []
     for block in blockchain.chain:
-        chain.append(block.toJSON())
+        chain.append(block.to_json())
     return jsonify(length=len(chain), chain=chain), HttpCode.OK.value
 
+@app.route('/addPeer', methods=['POST'])
+def add_peer():
+    data = request.get_json()
+    try:
+        peer = data["peer"]
+        peers.append(peer)
+    except KeyError:
+        IncorrectPayloadException()
+
+    return jsonify(success=True), HttpCode.OK.value
+
+
+@app.route('/wallet/all', methods=['GET'])
+def get_wallets():
+    wallets = []
+    for wallet, amount in blockchain.wallets.items():
```

Comment:
Do you need the `.items()` here? I would've thought just `blockchain.wallets` would work

Reply:
just wallets gives the keys for the dictionaries and not the key value pairs

Reply:
That surprises me as I am pretty sure I have used the following loop in the past to iterate over a dictionary:

`for key, value in dict:`

Reply:
![image](https://user-images.githubusercontent.com/57554901/113663402-975e2d80-966f-11eb-977e-7be2189639c7.png)

Reply:
It is def more intuitive but doesn't work

Reply:
Huh I'm wrong af, idk what I am thinking of then. Resolved